server/test/services/export/AbstractExporterTest.java

```diff
@@ -532,16 +534,36 @@ FakeProgramBuilder withQuestion(Question question) {
     }
 
     FakeProgramBuilder withHouseholdMembersEnumeratorQuestion() {
-      fakeProgramBuilder
-          .withBlock()
-          .withRequiredQuestion(testQuestionBank.applicantHouseholdMembers())
-          .withRepeatedBlock()
-          .withRequiredQuestion(testQuestionBank.applicantHouseholdMemberFavoriteShape())
-          .build();
+      addEnumeratorQuestion = true;
+      return this;
+    }
+
+    FakeProgramBuilder withHouseholdMembersJobsNestedEnumeratorQuestion() {
+      addNestedEnumeratorQuestion = true;
       return this;
     }
 
     Program build() {
+      if (addEnumeratorQuestion && addNestedEnumeratorQuestion) {
```

Comment:
Building these at the end, instead of as the builder is built, allows us to call both methods at the callsites in the tests, which makes it easier to tell what the test should be testing.